.travis.yml

```diff
@@ -23,6 +23,8 @@ before_script:
 script:
     # flake8 the diff
     - "git diff HEAD^ '*.py' | flake8 --diff --config flake8.cfg"
+    # add secret key to django settings.py
+    - echo "SECRET_KEY= '$(cat /dev/urandom | tr -dc 'a-z0-9!@#$%^&*(-_=+)' | head -c 50)'" >> platform/pulpcore/app/settings.py
```

Comment:
Nice one liner. I tested it. My only question is: should this this be added as an entry to server.yaml instead? It's Travis so it probably doesn't matter, but that is what we recommend the users to do. What do you think?